Translated/Tutorials/tut_006222.csv

```diff
@@ -0,0 +1,2 @@
+Title#0,"""Item Pack"""
+text0001#0,"""“In "Item Pack" in the "Item" menu,<br>you can check, use, and organize items.<br>You can check the useage and ability of an item in the "Item Details" window.<br><br>By placing the cursor on "Item Pack" at the top of the item menu,<br><pict Alpha_rightleft>you can switch the displayed contents of each storage."""
```

Comment:
you need to use `\""` for quotes

Reply:
Sorry so, will it be,  text0001#0,"""\“In "Item Pack" in the "Item" menu,<br>you~"

I'm still unsure entirely how to use GitHub.

Reply:
No, it would look like this:
`text0001#0,"""“In "Item Pack" in the \""Item\"" menu,<br>you can check, use, and organize items.<br>You can check, use, and organize items.<br>You can check the useage and ability of an item in the \""Item Details\"" window.<br><br>By placing the cursor on \""Item Pack\"" at the top of the item menu,<br><pict Alpha_rightleft>you can switch the displayed contents of each storage."""`

Reply:
Alright. Thank you very much for your help.

Reply:
Look at our Wiki for this: https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki#user-content-reminders-

To use quotes inside strings, use `\""`. For example: `"ARKS stands for \""ARTIFICIAL RELICT TO KEEP SPECIES\"". Isn't that neat?"`